libraries/botbuilder-dialogs/src/skillDialog.ts

```diff
@@ -174,6 +219,10 @@ export class SkillDialog extends Dialog<Partial<BeginSkillDialogOptions>> {
         return JSON.parse(JSON.stringify(activity));
     }
 
+    /**
+     * @private
```

Comment:
For private methods, if we are not adding any info we should leave only the `@private` tag